src/app/services/coffeeShops.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse, HttpHeaders } from '@angular/common/http';
+import { catchError, Observable, throwError } from 'rxjs';
+import { NGXLogger } from 'ngx-logger';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CoffeeShopsService {
+
+  constructor(private http: HttpClient,
+              private ngxLogger: NGXLogger) {
+  }
+
+  getToken(): Observable<any> {
+    return this.http.post(`https://blue-bottle-api-test.herokuapp.com/v1/tokens`, {});
+  }
+
+  getCoffeeShops(token: string): Observable<any> {
```

Comment:
Do you see using `any` as an ok/good practice in general?

Reply:
No, I am ashamed I left this in.